src/Client.php

```diff
@@ -107,10 +109,11 @@ public static function make($apiKey = null, $endpoint = null, $defaults = true)
      * @param \Bugsnag\Configuration                  $config
      * @param \Bugsnag\Request\ResolverInterface|null $resolver
      * @param \GuzzleHttp\ClientInterface|null        $guzzle
+     * @param ShutdownStrategyInterface               $shutdownStrategy
```

Comment:
Missing `|null`.

Reply:
Also, should use a FQCN out of convention.

Reply:
Fixed, thanks.